Storage/SQL/SQLiteLogins.swift

```diff
@@ -651,7 +651,7 @@ extension SQLiteLogins: SyncableLogins {
             ", sync_status = \(SyncStatus.Changed.rawValue) " +
         " WHERE guid = ?"
 
-        return self.db.run(update, withArgs: args)
+        return self.db.run(update, withArgs: args) >>> effect(self.notifyLoginDidChange)
```

Comment:
These lines will be called at least once for every login that changes during a sync, which is probably not what you want. Instead, call this *once* at the end of `LoginsSynchronizer.synchronizeLocalLogins`.